src/tests/dispatch-payload.test.ts

```diff
@@ -151,9 +159,12 @@ Deno.test("DispatchPayload function file compatibility tests", async (t) => {
     },
   );
   Deno.chdir(origDir);
+  mockFetch.uninstall();
 });
 
 Deno.test("DispatchPayload with unhandled events", async (t) => {
+  mockFetch.install();
```

Comment:
Any places where I added `mockFetch.install()` in this PR was due to running the tests without wifi turned on (I was on a plane). So we were doing network calls to the `complete*` APIs prior to this PR every time we ran the tests, locally or in CI 😬